index.html

```diff
@@ -20,7 +20,11 @@ <h1>Welcome to the student showcase!</h1>
     <div class="content">
         <div>ONE</div>
         <div>TWO</div>
-        <div>THREE</div>
+        <div><img src="https://imgur.com/a/isPUB" alt="That's Kevin.">
+          Kevin T Heinz
```

Comment:
Remember that HTML is whitespace agnostic... these carriage returns/newlines will all render on one line. Maybe divide these with `p` tags?